src/RoleObjects/AccessibilityObject.js

```diff
@@ -710,4 +710,18 @@ export default class AccessibilityObject {
       evt.preventDefault();
     }
   }
+  /**
+   * Event listener for valueChanged events
+   * @access private
+   * @param {SyntheticEvent} evt - React event
+   */
+  _onValueChanged(evt) {
```

Comment:
Why is this needed?  This is something I would only a subset of interactive roles to have.

Reply:
removed

Reply:
you might need to push your local branch.  I'm still seeing this function in the PR.